.github/actions/build_apko/action.yml

```diff
@@ -0,0 +1,61 @@
+name: Build apko image
+description: Build an apko image based on a supplied .yaml file
+
+inputs:
+  apkoConfig:
+    description: "Path to the apko .yaml config file. If left empty, all images will be built."
+    required: false
+  registry:
+    description: "Container registry to use"
+    default: "ghcr.io"
+    required: true
+  githubToken:
+    description: "GitHub authorization token"
+    required: true
+
+# Linux runner only (podman required)
```

Comment:
Maybe this limitation should be stated in the description? It's the first thing I read when I use an Action I have not used before.

Reply:
Due to issues with the `ghcr.io` registry I decided to roll back to docker, so I think this is obsolete.